src/content/docs/new-relic-one/use-new-relic-one/core-concepts/new-relic-explorer-view-performance-across-apps-services-hosts.mdx

```diff
@@ -66,23 +66,24 @@ The New Relic Explorer brings together data reported by any [entity](/docs/new-r
   You can create new entity types to monitor any data source. Learn more about [entity synthesis](https://github.com/newrelic/entity-definitions#entity-definitions).
 </Callout>
 
-To use the New Relic Explorer: Go to **[one.newrelic.com](http://one.newrelic.com)** and select **Explorer**.
+Use the New Relic Explorer at **[one.newrelic.com](http://one.newrelic.com)**!
 
-![nrexplorer2.png](./images/nrexplorer2.png "nrexplorer2.png")
+![nrexplorer2.png](./images/nrexplorer20210722.png "nrexplorer2.png")
```

Comment:
style guide recommends a good description, not just the filename. see the example in https://docs.newrelic.com/docs/style-guide/processes-procedures/embed-images/#embed

Reply:
Done! I always forget this!